src/main/Extensions/ApplicationSearch/Windows/WindowsApplicationRepository.ts

```diff
@@ -17,11 +18,26 @@ export class WindowsApplicationRepository implements ApplicationRepository {
     public constructor(
         private readonly powershellUtility: PowershellUtility,
         private readonly settings: Settings,
+        private readonly fileSystemUtility: FileSystemUtility,
         private readonly fileImageGenerator: FileImageGenerator,
         private readonly logger: Logger,
         private readonly assetPathResolver: AssetPathResolver,
     ) {}
 
+    private getExistingFolderPaths(candidates: string[]): string[] {
+        const existingFolderPaths: string[] = [];
+        for (const candidate of candidates) {
+            if (this.fileSystemUtility.isDirectory(candidate)) {
+                existingFolderPaths.push(candidate);
+            } else {
+                this.logger.warn(
+                    `Unable to get applications from folder "${candidate}". Reason: path doesn't exist or isn't a folder`,
+                );
+            }
+        }
+        return existingFolderPaths;
```

Comment:
We try to write code that is pleasant to read and easy on the eye. In my opinion this looks just too dense. Please add some space around the `for` loop:

```suggestion
        const existingFolderPaths: string[] = [];

        for (const candidate of candidates) {
            if (this.fileSystemUtility.isDirectory(candidate)) {
                existingFolderPaths.push(candidate);
            } else {
                this.logger.warn(
                    `Unable to get applications from folder "${candidate}". Reason: path doesn't exist or isn't a folder`,
                );
            }
        }

        return existingFolderPaths;
```

Reply:
Ok will check eslint/prettier to format that correctly.

Reply:
Prettier does not seem to complain about current code style but I have adjusted based on your comment. You might need to adapt your rules so contributors are not confused in the future.

Reply:
I'll add a eslint rule in a follow-up